src/main/java/com/google/api/codegen/util/NameFormatter.java

```diff
@@ -26,24 +26,29 @@
   String className(Name name);
 
   /**
-   * Formats the name as a variable name.
+   * Formats the name as a public variable name.
    */
-  String varName(Name name);
+  String publicVarName(Name name);
+
+  /**
+   * Formats the name as a private variable name.
+   */
+  String privateVarName(Name name);
```

Comment:
Ok, after seeing the usage, and also after seeing Chris' C# PR, I'd like to make a further tweak. Specifically, C# distinguishes between local variables (lower camel) and private fields (underscore + lower camel). So, I think it should look like this:

* publicVarName -> publicFieldName
* privateVarName ->
  1. localVarName (for parameters and local variables; "private" is confusing here)
  2. privateFieldName (for member variables in classes)